stackdio/salt/extmods/fileserver/stackdio.py

```diff
@@ -91,32 +80,26 @@ def _get_env_dir(saltenv):
     formula_dir = os.path.join(root_dir, 'formulas')
 
     if not os.path.exists(formula_dir):
-        os.mkdir(formula_dir, 0755)
+        os.mkdir(formula_dir, 0o755)
```

Comment:
yup, octal mode.  `0755` is only valid in python2, `0o755` is good in python 2 and 3